ecosystem_integration_templates/Databricks & Datarobot - Large Scale Forecasting/Databricks & Datarobot - Large Scale Forecasting.config.yaml

```
@@ -0,0 +1,12 @@
+---
+title: End-to-end time series workflow with DataRobot and Databricks
+file_name: Databricks & Datarobot - Large Scale Forecasting.ipynb
+description: |-
+  This accelerator is developed for use with Databricks to help users leverage the power of DataRobot for time-series modeling within their Databricks ecosystem.
+languages:
+  - python
+maintainers:
+  - Austin Chou 
+maintainers_email:
+  - austin.chou@datarobot.com 
+tags: []
```

```suggestion
tags: []
smoke_test:
  run_smoke_test: false
```
we have new fields as part of the `make accelerator` command